pkg/printers/internalversion/printers.go

```diff
@@ -962,27 +963,33 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 					lastRestartDate = terminatedDate
 				}
 			}
-			if container.State.Waiting != nil && container.State.Waiting.Reason != "" {
+			switch {
+			case container.State.Waiting != nil && len(container.State.Waiting.Reason) > 0:
```

Comment:
```suggestion
			case container.State.Waiting != nil && container.State.Waiting.Reason != "":
```
Please keep the method of checking for an empty string as it improves readability by making clear to the reader the value type.  Also, it maintains consistency with the rest of the code.

Reply:
Done